modules/op-scoping-github-token.adoc

```diff
@@ -0,0 +1,103 @@
+// This module is included in the following assembly:
+//
+// *cicd/pipelines/using-pipelines-as-code.adoc
+
+:_content-type: PROCEDURE
+[id="scoping-github-token_{context}"]
+= Scoping the GitHub token to additional repositories
+
+{pac} uses the GitHub App to generate a GitHub access token. It uses this token both for retrieving the pipeline payload from the repository and for interactions of the CI/CD process with GitHub repositories.
+
+By default, this token is scoped only to the repository where the payload comes from.
+However, in some cases, the developer team might want the token to allow control over additional repositories. For example, there might be a CI repository where the `.tekton/pr.yaml` file and source payload might be located, however the build process defined in `pr.yaml` might fetch tasks from a separate private CD repository.
```

Comment:
```suggestion
{pac} uses the GitHub app to generate a GitHub access token. This token is used to retrieve the pipeline payload from the repository, and to allow CI/CD processes to interact with GitHub repositories. By default, this token is scoped only to the repository where the payload comes from.

In some cases, you might want the token to have access to additional repositories. For example, there might be a CI repository where the `.tekton/pr.yaml` file and source payload are located, but the build process defined in `pr.yaml` needs to fetch tasks from a separate private CD repository.
```

suggestions

Reply:
I don't think adding passive voice is good, but did some rewriting.